Tests/JWTTests/Helper/throwsError.swift

```diff
@@ -1,10 +1,12 @@
 @testable import JWT
 import XCTest
 
-func assert<E: Error>(_ expression: @autoclosure () throws -> Any,
+func assert<E: Error>(
```

Comment:
For some reason it started throwing ambiguous type errors and I couldn't figure out why >_<. I did spend a while trying to debug it I promise haha. I'm not really too sure how `@autoclosure` is working.